thirdparty/protobuf/CMakeLists.txt

```diff
@@ -0,0 +1,125 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#------------------------------------------------------------------------------
+# Configure and install Google Protobuf ...
+#------------------------------------------------------------------------------
+
+set(PUSH_CMAKE_INTERPROCEDURAL_OPTIMIZATION_RELEASE "${CMAKE_INTERPROCEDURAL_OPTIMIZATION_RELEASE}")
+set(CMAKE_INTERPROCEDURAL_OPTIMIZATION_RELEASE OFF)
+
+if (MSVC)
+    set(protobuf_MSVC_STATIC_RUNTIME OFF CACHE BOOL "")
+endif()
+
+if(CMAKE_CROSSCOMPILING)
+    find_program(SYSTEM_PROTOC protoc PATHS ENV PATH)
+
+    if(SYSTEM_PROTOC)
+        execute_process(
+            COMMAND ${SYSTEM_PROTOC} --version
+            OUTPUT_VARIABLE PROTOC_VERSION
+            OUTPUT_STRIP_TRAILING_WHITESPACE
+        )
+
+        string(REPLACE " " ";" PROTOC_VERSION ${PROTOC_VERSION})
+        list(GET PROTOC_VERSION -1 PROTOC_VERSION)
+
+        message("Detected system protoc version: ${PROTOC_VERSION}")
+    else()
+        message(FATAL_ERROR "System Protobuf is needed while cross-compiling")
+    endif()
+
+    set(protobuf_BUILD_PROTOC_BINARIES OFF CACHE BOOL "Build libprotoc and protoc compiler" FORCE)
+endif()
+
+if (CMAKE_GENERATOR STREQUAL "Ninja")
+    set(MAKE_UTIL make)
+else()
+    set(MAKE_UTIL $(MAKE))
+endif()
```

Comment:
is `MAKE_UTIL` used?

Reply:
It can by used by ninja internally, I moved it here because it was present in the previous location.
But I'm not 100% sure about that.